stores/sql/sqlite/migrations/main/migration_00011_host_subnets.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `hosts` ADD COLUMN `subnets` text NOT NULL DEFAULT '';
```

Comment:
Ran into this on my node. Seems like SQLite has a default value but MySQL doesn't.

```
panic: failed to sync: failed to process updates: transaction failed (attempt 1): failed to apply chain updates: failed to update host: failed to insert host: Error 1364 (HY000): Field 'subnets' doesn't have a default value

goroutine 16 [running]:
go.uber.org/zap/zapcore.CheckWriteAction.OnWrite(0x0?, 0x0?, {0x0?, 0x0?, 0xc002748380?})
        /go/pkg/mod/go.uber.org/zap@v1.27.0/zapcore/entry.go:196 +0x54
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0039ae4e0, {0x0, 0x0, 0x0})
        /go/pkg/mod/go.uber.org/zap@v1.27.0/zapcore/entry.go:262 +0x3ec
go.uber.org/zap.(*SugaredLogger).log(0xc00007c3e8, 0x4, {0x112543d?, 0x0?}, {0xc0006d5f88?, 0x0?, 0xc00003e0c0?}, {0x0, 0x0, 0x0})
        /go/pkg/mod/go.uber.org/zap@v1.27.0/sugar.go:355 +0xec
go.uber.org/zap.(*SugaredLogger).Panicf(...)
        /go/pkg/mod/go.uber.org/zap@v1.27.0/sugar.go:229
go.sia.tech/renterd/internal/chain.(*ChainSubscriber).Run.func1()
        /renterd/internal/chain/chainsubscriber.go:145 +0x198
created by go.sia.tech/renterd/internal/chain.(*ChainSubscriber).Run in goroutine 1
        /renterd/internal/chain/chainsubscriber.go:132 +0x69
```

Reply:
Hadn't tested it yet on my nodes I'm still stuck with a `panic`. I'm syncing now on a fresh node though. I updated the column to be a `VARCHAR`. It's odd because I ran into the same panic and I thought I got around it by simply deleting the `DEFAULT ''` for MySQL. I only used `text` to stay somewhat consistent with the existing schema but a `VARCHAR` is actually better here for multiple reasons. I chose the (rather default) 255 length, it's at most ~80 characters so we should be good there.